xsd/UFTP-agr-dso.xsd

```diff
@@ -387,6 +415,11 @@
                         <xs:documentation>Reference to the concerning bilateral contract, if applicable.</xs:documentation>
                     </xs:annotation>
                 </xs:attribute>
+                <xs:attribute name="ServiceType" type="xs:string" use="optional">
+                    <xs:annotation>
+                        <xs:documentation>Service type for this request, the service type determines response characteristics such as latency or asset participation type.</xs:documentation>
```

Comment:
`
Service type for this request. The service type determines response characteristics such as latency and asset participation type.
`